resources/views/components/input-field-mobile.blade.php

```diff
@@ -0,0 +1,29 @@
+@props([
+    "type" => "text",
+    "id" => null,
+    "placeholder" => "",
+    "variant" => "primary"
```

Comment:
```suggestion
    "variant" => "textarea"
```
Odd name choices, could be more clear.

Reply:
changed to ``"variant" => "input"`` because textarea is not the default variant